run-tests.sh

```diff
@@ -45,3 +45,6 @@ docker run --rm \
 	-v ${kafka_repo_root}:${kafka_repo_root} \
 	${DOCKER_IMAGE} \
 	bash -c ${kafka_repo_root}/tests/run.sh
+
+cd ${kafka_repo_root}/kuttl-tests
```

Comment:
let's run first kuttl tests even before creating the big cluster? we are running them in pre-checks in case of Cassandra